### ISSUE TITLE
Inference

### ISSUE THREAD

**Issue by tommyshelby4** [garvita-tiwari/PoseNDF#28]
If I want to use the model just for inference and not from training, is the dataset generation necessary?

**Comment by tommyshelby4** [comment]
Additionally, how can I obtain the data, pose manifold, configuration file and PoseNDF checkpoint file for this setting **amass_flip_test**?

**Comment by garvita-tiwari** [comment]
> If I want to use the model just for inference and not from training, is the dataset generation necessary?

No the dataset generation is not necessary. You will just need to convert axis-angle poses to quaternion. And apply quaternion flip and normalization before feeding it to network. See sample_poses.py/L69:

` noisy_poses, _ = quat_flip(noisy_poses)`
 `noisy_poses = torch.nn.functional.normalize(noisy_poses,dim=-1)`






> Additionally, how can I obtain the data, pose manifold, configuration file and PoseNDF checkpoint file for this setting **amass_flip_test**?

This is the checkpoint and config file:https://nextcloud.mpi-klsb.mpg.de/index.php/s/4zxN93WL769pSAK?path=%2Fversion2
